examples/SPDXJSONExample-v2.0.json

```diff
@@ -2,380 +2,267 @@
+      "licenseId" : "LicenseRef-1"
     } ],
-    "annotations" : [ {
-      "annotator" : "Person: Jane Doe ()",
+    "annotation" : [ {
       "annotationDate" : "2010-01-29T18:30:22Z",
       "annotationType" : "OTHER",
+      "annotator" : "Person: Jane Doe ()",
       "comment" : "Document level annotation"
-    } ],
-    "reviewers" : [ {
-      "reviewer" : "Person: Suzanne Reviewer",
-      "reviewDate" : "2011-03-13T00:00:00Z",
-      "comment" : "Another example reviewer."
     }, {
-      "reviewer" : "Person: Joe Reviewer",
-      "reviewDate" : "2010-02-10T00:00:00Z",
+      "annotationDate" : "2010-02-10T00:00:00Z",
+      "annotationType" : "REVIEW",
+      "annotator" : "Person: Joe Reviewer",
       "comment" : "This is just an example.  Some of the non-standard licenses look like they are actually BSD 3 clause licenses"
+    }, {
+      "annotationDate" : "2011-03-13T00:00:00Z",
+      "annotationType" : "REVIEW",
+      "annotator" : "Person: Suzanne Reviewer",
+      "comment" : "Another example reviewer."
+    } ],
+    "documentDescribes" : [ "SPDXRef-File", "SPDXRef-Package" ],
+    "packages" : [ {
+      "comment" : "This package was converted from a DOAP Project by the same name",
+      "copyrightText" : "NOASSERTION",
+      "downloadLocation" : "NOASSERTION",
+      "filesAnalyzed" : false,
+      "homepage" : "http://www.openjena.org/",
+      "licenseConcluded" : "NOASSERTION",
+      "licenseDeclared" : "NOASSERTION",
+      "name" : "Jena"
+    }, {
+      "comment" : "This package was converted from a DOAP Project by the same name",
+      "copyrightText" : "NOASSERTION",
+      "downloadLocation" : "NOASSERTION",
+      "filesAnalyzed" : false,
+      "homepage" : "http://commons.apache.org/proper/commons-lang/",
+      "licenseConcluded" : "NOASSERTION",
+      "licenseDeclared" : "NOASSERTION",
+      "name" : "Apache Commons Lang"
+    }, {
+      "checksum" : [ {
+        "algorithm" : "SHA1",
+        "checksumValue" : "85ed0817af83a24ad8da68c2b5094de69833983c"
+      } ],
+      "description" : "The Saxon package is a collection of tools for processing XML documents.",
+      "downloadLocation" : "https://sourceforge.net/projects/saxon/files/Saxon-B/8.8.0.7/saxonb8-8-0-7j.zip/download",
+      "filesAnalyzed" : "false",
+      "homepage" : "http://saxon.sourceforge.net/",
+      "licenseComments" : "Other versions available for a commercial license",
+      "licenseConcluded" : "MPL-1.0",
+      "licenseDeclared" : "MPL-1.0",
+      "name" : "Saxon",
+      "packageFileName" : "saxonB-8.8.zip",
+      "versionInfo" : "8.8"
+    }, {
+      "annotation" : [ {
```

Comment:
Same question here as above -- should the package-level annotations array be plural rather than singular?